TeaLeaf/src/com/tealeaf/TeaLeaf.java

```diff
@@ -362,6 +362,23 @@ public void onGlobalLayout(){
 		});
 	}
 
+	private boolean isNotchedDevice() {
+		String[] notched_models = { "ANE-AL00", "ANE-L21", "ANE-TL00", "ANE-LX1", "ANE-LX2", "ANE-LX2J", "ANE-LX3",
+				"CLT-L04", "CLT-L09", "CLT-L29", "EML-L09", "EML-L22", "EML-L29", "ANE-L02", "ANE-L02K", "ANE-L03",
+				"ANE-L12JPZ", "ANE-L21", "ANE-L22", "ANE-L23", "ZE620KL", "ZS620KL", "COL-AL00", "COL-AL10", "COL-TL00",
+				"COL-TL10", "R15 Pro", "PAAM00", "R15", "CPH1819", "V9", "X21", "X21 UD", "A6000", "A6003", "LM-G710",
+				"G710" };
```

Comment:
can you put a comment against each block, so we know what kind of devices these are. Will be useful to edit/add in the future

Reply:
Sure.